### PR TITLE
use better defaults for ACE

### PR DESCRIPTION
**Pull request by mulby**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).

FYI @thallada @ormsbee 

NOTE: Depends on https://github.com/edx/edx-platform/pull/16238

### CODE REVIEW

**Review by nasthagiri — APPROVED** [review]

**Review by sandroroux — APPROVED** [review]

**Comment by mulby** [comment]
@edx/devops can one of you review this?

File: playbooks/roles/edxapp/defaults/main.yml [hunk 1]
```diff
@@ -237,10 +237,10 @@ EDXAPP_SOCIAL_AUTH_OAUTH_SECRETS: ""
 
 EDXAPP_ACE_CHANNEL_SAILTHRU_API_KEY: ""
 EDXAPP_ACE_CHANNEL_SAILTHRU_API_SECRET: ""
-EDXAPP_ACE_ENABLED_CHANNELS: []
-EDXAPP_ACE_ENABLED_POLICIES: []
+EDXAPP_ACE_ENABLED_CHANNELS: ["file_email"]
```
Comment (feanil) [inline playbooks/roles/edxapp/defaults/main.yml:240]: How do I know what can be in these lists?  Can you link to the relevant code or docs?

File: playbooks/roles/edxapp/defaults/main.yml [hunk 2]
```diff
@@ -237,10 +237,10 @@ EDXAPP_SOCIAL_AUTH_OAUTH_SECRETS: ""
...
+EDXAPP_ACE_ENABLED_POLICIES: ["bulk_email_optout"]
 EDXAPP_ACE_CHANNEL_SAILTHRU_DEBUG: True
-EDXAPP_ACE_CHANNEL_SAILTHRU_TEMPLATE_NAME: !!null
+EDXAPP_ACE_CHANNEL_SAILTHRU_TEMPLATE_NAME: ""
```
Comment (feanil) [inline playbooks/roles/edxapp/defaults/main.yml:243]: NIT: Can you document why null and empty string have different behaviors and what those behaviors are?

**Comment by ormsbee** [comment]
Closing this year old PR. Please feel free to reopen if appropriate.

**Pull request closed without merging**